services/kodo/service.go

```diff
@@ -53,18 +56,20 @@ func (s *Service) list(ctx context.Context, opt pairServiceList) (it *typ.Storag
 }
 
 func (s *Service) nextStoragePage(ctx context.Context, page *typ.StoragerPage) error {
-	buckets, err := s.service.Buckets(false)
+	buckets, err := s.service.Buckets(true)
```

Comment:
Why we need to modify the `shared` to `true` here?

Reply:
In fact, it's mistake.